WalletLibrary/WalletLibraryTests/Networking/Encoders/FormURLEncodedRequestEncoderTests.swift

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import XCTest
+@testable import WalletLibrary
+
+class FormURLEncodedRequestEncoderTests: XCTestCase
+{
+    func testEncode_WithValidRequest_EncodesRequest() throws
```

Comment:
Could you add negative test cases in this scenario?

Reply:
Unfortunately there isn't really a good way to test the negative case, the  encodedString.data(using: .ascii) will never really return nil in this particular case.